installers/agent/release/agent.sh

```diff
@@ -121,5 +121,5 @@ if [ "$DAEMON" == "Y" ]; then
     eval "nohup $CMD >>$LOG_FILE 2>&1 &"
```

Comment:
Can you replace this with exec as well? You may need to do `exec setsid nohup...`?